string robot_name
---
bool success
mcs_goal_pose[] goals